monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,115 @@
+[id="configuring-the-monitoring-stack"]
+= Configuring the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: configuring-the-monitoring-stack
+
+toc::[]
+
+The {product-title} 4 installation program provides only a low number of configuration options before installation. Configuring most {product-title} framework components, including the cluster monitoring stack, happens post-installation.
+
+This section explains what configuration is supported, shows how to configure the monitoring stack, and demonstrates several common configuration scenarios.
+
+== Prerequisites
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/scaling-cluster-monitoring-operator.html[Scaling the Cluster Monitoring Operator] and verify that you have sufficient resources.
```

Comment:
This is a link that directs the reader to the OpenShift Container Platform documentation. Is this relevant to ROSA, or should this link be removed? The reader might not realise that they are being directed to the docs for a different platform type.

Reply:
Keep.